cloudkeeper-core/cloudkeeper-linker/src/main/java/com/svbio/cloudkeeper/linker/FinishContext.java

```diff
@@ -21,6 +21,9 @@
 import java.util.Optional;
 
 final class FinishContext {
+    public static final Name CLOUD_KEEPER_ELEMENT_REFERENCE_NAME
+        = Name.qualifiedName(CloudKeeperElementReference.class.getName());
+    public static final Name OBJECT_NAME = Name.qualifiedName(Object.class.getName());
```

Comment:
Change not really related to the style guide, so I would have left as-is. But if you do, replace `public` by `private`.